pimgen.py

```diff
@@ -571,7 +571,8 @@ def get_interfaces(targetdir):
 
         for y in yaml_files:
             # parse only phosphor dbus related interface files
-            if not (y.startswith('xyz') or y.startswith('com/ibm/ipzvpd')):
+            if not (y.startswith('xyz') or y.startswith('com/ibm/ipzvpd')
+                    or y.startswith('com/ibm/Control/Host')):
```

Comment:
Can this be just startswith('com/ibm')? 
If there are any other/new interfaces added in the future under 'com/ibm' we will face the same issue I think...